TestAnimals/TestAnimals.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The typical structure for the test projects is as following:
```
- src
   - <projects go here>
- test
   -  <test projects go here>
   - ```

The naming conventions are: `<project name>.UnitTests or <project name>.IntegrationTests or <project name>.Tests `

In this case if project is `AnimalsTest`, the unit test project should be `AnimalTest.UnitTests`